src/main.rs

```diff
@@ -1,30 +1,30 @@
 // A terrible dependency-free hack.
-fn rand() -> u64 {
-    let now = std::time::SystemTime::now();
-    let some_number = now.duration_since(std::time::UNIX_EPOCH).unwrap().as_millis() as u64;
-	let mut xorshift = some_number;
-	xorshift ^= xorshift << 13;
-	xorshift ^= xorshift >> 7;
-	xorshift ^= xorshift << 17;
-    return xorshift;
+fn rand() -> u8 {
+    unsafe {
+      *(&std::time::Instant::now() as *const std::time::Instant as *const u8)
```

Comment:
Not that the original was particularly high-quality, but my only concern here is that some time APIs use time values where the low-order bits have low entropy (or are possibly even fixed). The original xorshift was done to mix some of the higher-order bits with the lower-order bits.